CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+## `Polkadot` projects is a **OPENISH Open Source Project**
+-----------------------------------------
+
+## What?
+
+Individuals making significant and valuable contributions are given commit-access to a project to contribute as they see fit. A project is more like an open wiki than a standard guarded open source project.
+
+## Rules
+
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+
+1. **No `--force` pushes** or modifying the Git history in any way.
+1. **Non-master branches**, prefixed with a short name moniker (e.g. `gav-my-feature`) must be used for ongoing work.
+1. **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+1. A pull-request *must not be merged until CI* has finished successfully.
+1. Contributors should adhere to the house coding style, as specific on the wiki.
```

Comment:
`s/specific/specifed/`?
May want to add the link: https://github.com/paritytech/polkadot/wiki/Style-Guide